pkgs/by-name/wi/wireguard-ui/package.nix

```
@@ -0,0 +1,80 @@
+{
+  buildGoModule,
+  lib,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  yarn,
+  fixup-yarn-lock,
+  nodejs,
+  fetchpatch2,
+}:
+
+buildGoModule rec {
+  pname = "wireguard-ui";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "ngoduykhanh";
+    repo = "wireguard-ui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-fK7l9I2s0zSxG1g1oQ1KjJZUcypwS9DxnNN7lhVI+1s=";
+  };
+
+  patches = [
+    (fetchpatch2 {
+      url = "https://github.com/ngoduykhanh/wireguard-ui/commit/2fdafd34ca6c8f7f1415a3a1d89498bb575a7171.patch";
```

Needs name and hash. While we're at it, we can use

```suggestion
      url = "https://github.com/ngoduykhanh/wireguard-ui/commit/2fdafd34ca6c8f7f1415a3a1d89498bb575a7171.patch?full_index=1";
```

Which is recommended practice.